core/modules/syncer.js

```diff
@@ -129,47 +133,40 @@ function Syncer(options) {
 	});
 }
 
+/*
+Return an array of the tiddler titles that are subjected to syncing
+*/
+Syncer.prototype.getSyncedTiddlers = function(source) {
+	return this.filterFn.call(this.wiki,source);
+};
+
 /*
 Read (or re-read) the latest tiddler info from the store
 */
 Syncer.prototype.readTiddlerInfo = function() {
-	// Hashmap by title of {revision:,changeCount:,adaptorInfo:,needsToBeLoaded:,hasBeenLazyLoaded:}
+	// Hashmap by title of {revision:,changeCount:,adaptorInfo:}
+	// "revision" is the revision of the tiddler last seen on the server, and "changecount" is the corresponding local changecount
 	this.tiddlerInfo = {};
 	// Record information for known tiddlers
 	var self = this,
```

Comment:
`var self` is also defined at the prototype root. ... Then it is redefined in the functions. This makes the code really hard to read and error-prone.

Reply:
Ouch! Where's the definition at the prototype root? Do you mean line 39? That variable is only local to the constructor.